common/src/main/java/net/william278/huskclaims/claim/Claim.java

```diff
@@ -140,6 +140,17 @@ public class Claim implements Highlightable {
     @SerializedName("inherit_parent")
     private boolean inheritParent;
 
+    /**
+     * If this is a child claim, whether to inherit member trust levels from the parent.
+     * <p>
+     * If set to false, this child claim will be restricted.
+     */
```

Comment:
Don't forget to update this JavaDoc comment :)

Reply:
Updated the JavaDoc :)